packages/serverless/src/apollo/app/transaction.ts

```diff
@@ -60,13 +79,28 @@ const enqueueTransaction = authorizeWrapper(
     )
 );
 
+const confirmTransaction = authorizeWrapper(
+  Joi.object({
+    databaseName,
+    transactionType,
+  }),
+  async (_root: unknown, args: TTransactionConfirmRequest, ctx) =>
+    confirmTransactionDomain(
+      args.databaseName,
+      ctx.userName,
+      args.transactionType,
+      args.txHash
+    )
```

Comment:
the third arg of confirmTransactionDomain required id not transactionType

Reply:
done